contrib/php8.3/template.py

```diff
@@ -244,6 +249,19 @@ def post_install(self):
                 outf.write(f"zend_extension={extso}\n")
             else:
                 outf.write(f"extension={extso}\n")
+    # remove temporary files/dirs that shouldn't be part of package
+    for f in [
+        ".channels",
+        ".depdb",
+        ".depdblock",
+        ".filemap",
+        ".lock",
+        f"usr/share/php{_majver}/pear/.channels",
+        f"usr/share/php{_majver}/pear/.filemap",
+        f"usr/share/php{_majver}/pear/.lock",
+        f"usr/share/php{_majver}/pear/.registry",
+    ]:
+        self.rm(f"{self.destdir}/{f}", recursive=True)
```

Comment:
```suggestion
        self.uninstall(f, recursive=True)
```

but is there a way to make them somehow not installed in the first place? if it's a small build system patch it's a bit easier to not miss it later

Reply:
I didn't find an easy way.  I also noticed inside pear .phar installer that the RPM spec file also removes these files after installation so I think it's the only way of doing that unfortunately

Reply:
that's ok :) uninstall is fine then